### ISSUE TITLE
OptimizerFactory not defined

### ISSUE THREAD

**Issue by diegozea** [englhardt/SVDD.jl#19]
Hi! I've just installed SVDD in Julia 1.4.2 (Ubuntu) and I found the following error:

```
julia> using SVDD
[ Info: Precompiling SVDD [83536b18-e0df-11e8-1e3f-f991e449cee4]
ERROR: LoadError: LoadError: UndefVarError: OptimizerFactory not defined
Stacktrace:
 [1] getproperty(::Module, ::Symbol) at ./Base.jl:26
 [2] top-level scope at /home/elin/.julia/packages/SVDD/Ed7xK/src/classifiers/sub/classifier_sub_svdd.jl:76
 [3] include(::Module, ::String) at ./Base.jl:377
 [4] include(::String) at /home/elin/.julia/packages/SVDD/Ed7xK/src/SVDD.jl:1
 [5] top-level scope at /home/elin/.julia/packages/SVDD/Ed7xK/src/SVDD.jl:8
 [6] include(::Module, ::String) at ./Base.jl:377
 [7] top-level scope at none:2
 [8] eval at ./boot.jl:331 [inlined]
 [9] eval(::Expr) at ./client.jl:449
 [10] top-level scope at ./none:3
in expression starting at /home/elin/.julia/packages/SVDD/Ed7xK/src/classifiers/sub/classifier_sub_svdd.jl:76
in expression starting at /home/elin/.julia/packages/SVDD/Ed7xK/src/SVDD.jl:8
ERROR: Failed to precompile SVDD [83536b18-e0df-11e8-1e3f-f991e449cee4] to /home/elin/.julia/compiled/v1.4/SVDD/B3Nr9_9qXKP.ji.
Stacktrace:
 [1] error(::String) at ./error.jl:33
 [2] compilecache(::Base.PkgId, ::String) at ./loading.jl:1272
 [3] _require(::Base.PkgId) at ./loading.jl:1029
 [4] require(::Base.PkgId) at ./loading.jl:927
 [5] require(::Module, ::Symbol) at ./loading.jl:922
```

Thanks in advance,

Best regards

**Comment by englhardt** [comment]
Hey, [JuMP](https://github.com/jump-dev/JuMP.jl) and [MathOptInterface](https://github.com/jump-dev/MathOptInterface.jl) changed how solvers are passed to the optimization problem in their most recent versions (JuMP `0.21.x`). We plan to update SVDD.jl to support the recent version of JuMP in the future.
For now, you can downgrade your JuMP to `0.20.1` by running `pkg> add JuMP@0.20.1` to fix this issue.

Edit: Fixed install command (use `JuMP@0.20.1` not `JuMP#0.20.1`).

**Comment by diegozea** [comment]
Great! Thank you very much!
